keyboards/xd75/keymaps/cryptokid91/keymap.c

```diff
@@ -0,0 +1,177 @@
+/* Copyright 2020 Marco Stålfors
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#include QMK_KEYBOARD_H
+
+// Custom keycodes for longer things
+#define MS_LW LT(_LOWER, KC_BSPC)
+#define MS_RS LT(_RAISE, KC_DEL)
+#define MS_US MT(MOD_RSFT, KC_UP)
+
+// Layer shorthand
+#define _QWERTY 0
+#define _LOWER  1
+#define _RAISE  2
+#define _RESET  3
```

Comment:
A better way: use an `enum` statement:

```c
enum layer_names {
    _QWERTY,
    _LOWER,
    _RAISE,
    _RESET
};
```

This makes it so you don't have to renumber all the indexes if you change your layers in future. This code block is identical to your current code once compiled.